src/app/rundown-view/components/segment/segment.component.html

```diff
@@ -7,8 +7,11 @@
     <div class="c-segment__header-toolbar"
       [class.c-segment__header-toolbar--unsynced]="segment.isUnsynced"
     >
-      <span class="segment-identifier">
-        <ng-container *ngIf="segment.isUnsynced">UNSYNCED</ng-container>
+      <span class="segment-reference-tag">
+        <span *ngIf="!segment.isUnsynced else unsynced">{{segment.referenceTag}}</span>
```

Comment:
Shouldn't the reference tag and unsynced information be available at the same time if both are present?

Reply:
It is with the new design!